abcclassroom/quickstart.py

```diff
@@ -0,0 +1,93 @@
+"""
+abc-classroom.dir-setup
+========================
+
+"""
+
+import os
+import shutil
+import argparse
+import abcclassroom
+
+
+def path_to_example(dataset):
+    """
+    Construct a file path to an example dataset.
+    This file defines helper functions to access data files in this directory,
+    to support examples. Adapted from the PySAL package.
+
+    Parameters
+    ----------
+    dataset: string
+        Name of a dataset to access (e.g., "sample_config.yml")
+            
+    Returns
+    -------
+    string
+        A file path (string) to the dataset
+    """
+    abcclassroom_path = os.path.split(abcclassroom.__file__)[0]
+    data_dir = os.path.join(abcclassroom_path, "example-data")
+    data_files = os.listdir(data_dir)
+    if dataset not in data_files:
+        raise KeyError(dataset + " not found in abc-classroom example data.")
+    return os.path.join(data_dir, dataset)
+
+
+def create_dir_struct():
+    """
+    Creates a directory structure that can be used to start an abc-classroom course. This includes a main directory,
+    two sub directories for templates and cloned files, and a sample configuration file.
+    """
+    # Creating the default names for the cloned and template repos.
+    course = "course_dir"
+    # Making sure the configuration file is where it's supposed to be.
+    config = path_to_example("sample_config.yml")
+    if not os.path.exists(config):
+        raise ValueError(
+            "Configuration file can't be located, please ensure abc-classroom has been installed correctly"
```

Comment:
here i think you want to specify the actual file. how about

Oops, we can't seem to find the config.yml configuration file. This file should be in the root of your course-name-here directory. Please ensure that you have all the needed  files

ok before i go any further, @nkorinek why would a user see this error? i'm a bit confused because don't we create it for them?

Reply:
@lwasser I was getting this error when it wasn't ran in the abc-classroom dir before I added `path_to_example`, so now it seems very unlikely. I could take it out for sure!

Reply:
@nkorinek it seems like it could be a good check but i'm just not sure what the message should say as i wasn't sure how it could occur from a user's perspective. 

@kcranston what do you think? it seems like a viable check but i wasn't sure when it might get triggered.